agent/ecscni/plugin_test.go

```diff
@@ -1,4 +1,4 @@
-// +build unit
+// +build linux,unit
```

Comment:
Does this mean that these test will run only on Linux?

Reply:
yes, task networking is not supported on windows. i don't see why these code paths need to be tested on windows.

Reply:
I think `agent/ecscni/plugin.go` itself is not linux specific, that's why we have the tests running in both. 
Can you add a note we need to change that as well and also the octokitten for windows that was updated and not needed anymore?